src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +141,39 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v1.2.1",
+     *       "lang": "java",
+     *       "lang_version": "oracle-v1.9.0",
+     *       "platform": "linux",
+     *       "platform_version": "10"
+     *     }
+     *   }
+     * }
+     *
+     * @param mixed    $requestPacket
+     *
+     * @return mixed
+     */
+    private function addMeta($requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => 'hardcoded for now',
+                'lang' => 'php',
+                'lang_version' => 'hmm',
+                'platform' => php_uname('s'),
+                'platform_version' => php_uname('v m')
```

Comment:
I took a liberty to add machine type as well as per http://php.net/manual/pl/function.php-uname.php. What do you think?

Reply:
It could be useful, though I think we'd probably be better having it in a separate key, such as `platform_arch`, so the parsing is easier on the other side.

Reply:
removed then for the time being, as it wasn't in the spec, we may add it any time, cheers